core/src/test/java/com/linecorp/armeria/internal/annotation/AnnotatedHttpServiceResponseConverterTest.java

```diff
@@ -319,6 +321,26 @@ public Object expectOk() {
                     return Flux.just("foo", "bar", "baz", "qux");
                 }
             });
+
+            sb.annotatedService("/event-stream", new Object() {
+                @Get("/stream")
+                @ProducesEventStream
```

Comment:
`@ResponseConverter(ServerSentEventResponseConverterFunction.class)` is not necessary anymore because #1560 has been merged. :-)